### ISSUE TITLE
Acorn crashed when deleting an entry-point node that was the root of all nodes in the entire LrL intersections tree

### ISSUE THREAD

**Issue by qubist** [lightningrodlabs/acorn#186]
```
Cannot read properties of undefined (reading 'outcomeActionHash')

TypeError: Cannot read properties of undefined (reading 'outcomeActionHash')
    at file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:390738
    at Array.forEach (<anonymous>)
    at ef (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:390611)
    at rf (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:392475)
    at n (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:437556)
    at file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:351:437566
    at us (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:83260)
    at Lc (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:102452)
    at t.unstable_runWithPriority (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:317:3844)
    at Ko (file:///Applications/Acorn.app/Contents/Resources/app.asar/web/app.js:309:45048)
```

I selected it and then pressed backspace

**Comment by Connoropolous** [comment]
I also just saw this behaviour when unmaking an Outcome as an entry point. Symptom was that it caused the Outcomes not to render on the map.

**Comment by Connoropolous** [comment]
The error message, which only appeared in the JS console was: 
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'outcomeActionHash')
    at drawEntryPoints.js:30:43
    at Array.forEach (<anonymous>)
    at drawEntryPoints (drawEntryPoints.js:29:21)
    at drawing_render (index.ts:177:5)
    at doRender (MapView.component.tsx:72:9)
    at dispatch (redux.js:297:1)
    at middleware.ts:73:14
    at step (hcWebsockets.ts:79:2)
    at Object.next (hcWebsockets.ts:79:2)
    at hcWebsockets.ts:79:2
```

**Comment by Connoropolous** [comment]
fixed in v2.1.0

**Comment by TechSupport26** [comment]
In v2.1.1-alpha when I remove a parent card all the other cards are invisible unless I refresh.

https://user-images.githubusercontent.com/65631811/199436963-168ebde7-bc8a-404f-af7a-b80f5f11eda0.mp4
